src/test/unit_tests/different_clients/test_go_sdkv2_script.js

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) 2025 NooBaa */
+'use strict';
+
+// disabling init_rand_seed as it takes longer than the actual test execution
+process.env.DISABLE_INIT_RANDOM_SEED = "true";
+
+// setup coretest first to prepare the env
+const coretest = require('../coretest');
+coretest.setup({ pools_to_create: [coretest.POOL_LIST[1]] });
```

Comment:
Can you explain in short, why `pool-with-10-hosts` and its usage?

Reply:
In this test, I'm using `name: 'pool-with-1-host'` (and not `name: 'pool-with-10-hosts'`), as I need a minimal check and I can use only 1 node. Probably tests that are focused on hosts will use more hosts for testing, in this case, it is basic.